### DISCUSSION TITLE
event needs to be extended for naturalevent

### DISCUSSION THREAD

**Discussion by aahmadai** [schemaorg/schemaorg#3241] [Ideas]
the thing -> event is missing the concept of a naturalevent type e.g like natural disasters

https://schema.org/Event

[More Specific Types](https://schema.org/Event#subtypes)
[BusinessEvent](https://schema.org/BusinessEvent)
[ChildrensEvent](https://schema.org/ChildrensEvent)
[ComedyEvent](https://schema.org/ComedyEvent)
[CourseInstance](https://schema.org/CourseInstance)
[DanceEvent](https://schema.org/DanceEvent)
[DeliveryEvent](https://schema.org/DeliveryEvent)
[EducationEvent](https://schema.org/EducationEvent)
[EventSeries](https://schema.org/EventSeries)
[ExhibitionEvent](https://schema.org/ExhibitionEvent)
[Festival](https://schema.org/Festival)
[FoodEvent](https://schema.org/FoodEvent)
[Hackathon](https://schema.org/Hackathon)
[LiteraryEvent](https://schema.org/LiteraryEvent)
[MusicEvent](https://schema.org/MusicEvent)
[PublicationEvent](https://schema.org/PublicationEvent)
[SaleEvent](https://schema.org/SaleEvent)
[ScreeningEvent](https://schema.org/ScreeningEvent)
[SocialEvent](https://schema.org/SocialEvent)
[SportsEvent](https://schema.org/SportsEvent)
[TheaterEvent](https://schema.org/TheaterEvent)
[VisualArtsEvent](https://schema.org/VisualArtsEvent)

**Comment by danbri** [comment]
What’s a natural event?

**Comment by aahmadai** [comment]
any event that occurs naturally like a natural disaster usually as a result of a natural agent

mudslide, avalanche, drought, earthquake, flood, hurricane, tornado, etc.

**Comment by danbri** [comment]
Climate change?

**Comment by aahmadai** [comment]
weather would be more a general term, climate would be more a specific term for weather for long-term change

**Comment by aahmadai** [comment]
climate change is not always a natural causation

**Comment by aahmadai** [comment]


**Reply by Jasperrr91** [reply]
I have no idea why this Github issue suddenly popped up in my mentions as I have never subscribed to this. However, I just saw 8 email notifications over the span of 20 minutes, of @aahmadai calling the maintainer a racist, speaking about making assumptions about people based on their name, discriminating contributors, "whites not taking non-whites serious", etcetera.

Why did you come to this conclusion @aahmadai ? I see an open source project with 699 open issues. It seems nothing but logical that your issue has not been fixed within a month. I also do not see any pull request from yourself where you try to fix this issue. If you urgently need this change, feel free to drop a pull request and check with the contributors if they can review it.

**Comment by aahmadai** [comment]


**Reply by Jasperrr91** [reply]
Why will you get blocked? Nobody get's blocked for contributing, calm down man.
